### DISCUSSION TITLE
<Payment /> error on debit card

### DISCUSSION THREAD

**Discussion by LucaCevasco** [mercadopago/sdk-react#42] [Q&A] [answered]
Hello! The payment brick fails when using a debit method. Credit card payment doesn't generate any issues. When selecting debit as the payment method, the following error pops up:
{
"type": "non_critical",
"cause": "missing_payment_information",
"message": "The payment method id is missing"
}

It's worth to say that the SDK version being used is 0.0.8 and the app is a clean React app using Vite

**Comment by AdaltonLeite** [comment]
Hi @LucaCevasco,

Can you please provide us with more details and maybe a sandbox of your code so we can identify the error?

**Comment by LucaCevasco** [comment]
Hello, I am unable to provide you with a sandbox environment for testing, but the bug can be reproduced in our production environment without incurring any charges. Regarding the error, I believe I understand the problem now. It appears that a user from Argentina attempted to pay with a prepaid card, but the payment brick did not recognize it as a debit card. Is it possible to extend the payment brick to include support for prepaid cards in the near future?

**Reply by AdaltonLeite** [reply]
Today we really don't offer this type of payment. 

But we already have in our backlog to enable the prepaid cards for PaymentBrick, soon it will be available. For now you can see the payment methods that this brick supports in this [documentation](https://www.mercadopago.com.ar/developers/es/docs/checkout-bricks/payment-brick/additional-customization/manage-payment-methods).

**Reply by LucaCevasco** [reply, marked as answer]
ok, thank you for the quick response.

**Comment by LucaCevasco** [comment]
Hello again, quick question... there is support for brubank card provider in PaymentBrick? we have an user reporting error with that debit card. I will get more info.

**Reply by AleFossati** [reply]
Hello Luca, I guess it should work fine. Did you manage to get more info about it?
